multi-agent/lib/mas/blueprints/service.py

```
@@ -157,20 +168,25 @@ def list_resolved_docs(
+        and to resolve each document's built-in overlays.
         """
         docs = self._repo.list_docs(identity=identity, **pg)
         resolved_docs = []
 
         for doc in docs:
             try:
-                resolved_docs.append(self._resolve_doc(doc))
+                resolved_docs.append(self._resolve_doc(doc, identity=identity))
             except Exception as e:
                 print(f"Skipping blueprint '{doc.blueprint_id}': resolution failed — {e}")
                 continue
```

Ruff (0.15.21)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)